docs/spark/spark-queries.md

```diff
@@ -175,45 +178,37 @@ For Spark 3, prior to 3.2, the Spark [session catalog](../spark-configuration#re
-| made_current_at         | snapshot_id         | parent_id           | is_current_ancestor |
-+-------------------------+---------------------+---------------------+---------------------+
+
+| made_current_at | snapshot_id  | parent_id | is_current_ancestor |
+| -- | -- | -- | -- |
 | 2019-02-08 03:29:51.215 | 5781947118336215154 | NULL                | true                |
 | 2019-02-08 03:47:55.948 | 5179299526185056830 | 5781947118336215154 | true                |
 | 2019-02-09 16:24:30.13  | 296410040247533544  | 5179299526185056830 | false               |
 | 2019-02-09 16:32:47.336 | 2999875608062437330 | 5179299526185056830 | true                |
 | 2019-02-09 19:42:03.919 | 8924558786060583479 | 2999875608062437330 | true                |
 | 2019-02-09 19:49:16.343 | 6536733823181975045 | 8924558786060583479 | true                |
-+-------------------------+---------------------+---------------------+---------------------+
-```
 
 {{< hint info >}}
 **This shows a commit that was rolled back.** The example has two snapshots with the same parent, and one is *not* an ancestor of the current table state.
 {{< /hint >}}
 
 ### Snapshots
 
-To show the valid snapshots for a table, run:
+To show the valid snapshots for a table:
 
 ```sql
 SELECT * FROM prod.db.table.snapshots
 ```
-```text
-+-------------------------+----------------+-----------+-----------+----------------------------------------------------+-------------------------------------------------------+
-| committed_at            | snapshot_id    | parent_id | operation | manifest_list                                      | summary                                               |
-+-------------------------+----------------+-----------+-----------+----------------------------------------------------+-------------------------------------------------------+
-| 2019-02-08 03:29:51.215 | 57897183625154 | null      | append    | s3://.../table/metadata/snap-57897183625154-1.avro | { added-records -> 2478404, total-records -> 2478404, |
-|                         |                |           |           |                                                    |   added-data-files -> 438, total-data-files -> 438,   |
-|                         |                |           |           |                                                    |   spark.app.id -> application_1520379288616_155055 }  |
+
+| committed_at | snapshot_id | parent_id | operation | manifest_list | summary |
+| -- | -- | -- | -- | -- | -- |
+| 2019-02-08 03:29:51.215 | 57897183625154 | null      | append    | s3://.../table/metadata/snap-57897183625154-1.avro | { added-records -> 2478404, total-records -> 2478404, added-data-files -> 438, total-data-files -> 438, spark.app.id -> application_1520379288616_155055 } |
 | ...                     | ...            | ...       | ...       | ...                                                | ...                                                   |
```

Comment:
This is not related to your change, but do you think we can remove this row of  ```...```, or populate it with more meaningful value ?

They look strange and I am not sure what purpose this serves.